cmd/boskosctl/boskosctl.go

```diff
@@ -100,8 +98,8 @@ scripts with a simple interface.`,
 		Args: cobra.NoArgs,
 	}
 	root.PersistentFlags().StringVar(&options.serverURL, "server-url", "", "URL of the Boskos server")
-	root.PersistentFlags().StringVar(&options.username, "username", "", "Username used to access the Boskos server")
-	root.PersistentFlags().StringVar(&options.passwordFile, "password-file", "", "The path to password file used to access the Boskos server")
+	root.PersistentFlags().MarkDeprecated("username", "this flag has been deprecated")
+	root.PersistentFlags().MarkDeprecated("password-file", "this flag has been deprecated")
```

Comment:
not respected at all is more than deprecated. why even leave the flag?

Reply:
If someone was using it in their deployments, boskos would still start instead of crashlooping.

Happy to get rid of the flags properly